spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -608,13 +608,50 @@
       end
     end
 
+    describe 'line 45 - balance of tax' do
+      let(:intake) { create(:state_file_nj_intake) }
+      it 'sets line 45 to calculated value' do
+        stub_value = 20_000
```

Comment:
This was a slight changed from us calling it `expected_value` in other tests, which felt weird to me because if we're stubbing it, it's not "expected" in the same way I think of it...